app/models/course.rb

```diff
@@ -117,6 +117,7 @@ def initialize_duplicate(duplicator, other)
     self.start_at += duplicator.time_shift
     self.end_at += duplicator.time_shift
     self.title = duplicator.new_course_title
+    self.registration_key = nil
```

Comment:
Actually, should we generate a new registration key or just set it to `nil` by default?